specs/patterns.md

```diff
@@ -542,9 +542,58 @@ None known.
 
 A [data resource][dr] where the `profile` is the same as that of a [data package][dp] or [identifier][dpi].
 
-This could be used as part of a "Data Package Catalog", probably specified as a `profile` like
-`data-package-catalog`,`tabular-data-package-catalog`, or `document-data-package-catalog` for restricting the resources to data packages.
+### Data Package Catalog
```

Comment:
I'd make this the main ## heading. I'd also suggest renaming e.g. to

"Describing Data Catalogs using the Data Package Format"